agenta-web/cypress/e2e/regex-evaluation.cy.ts

```diff
@@ -0,0 +1,148 @@
+import {app_id} from "../support/commands/utils"
+
+describe("Regex Evaluation workflow", () => {
+    context("When creating variants and testsets", () => {
+        it("Should successfully create app variant and testset", () => {
+            cy.createVariantsAndTestsets()
```

Comment:
Since, this is not a test but a pre test routing, please wrap it inside before() clause